frontend/src/app/organization/organization-details/organization-details.component.ts

```diff
@@ -1,7 +1,8 @@
-import { Component, Input } from '@angular/core';
+import {Component, Inject, Input} from '@angular/core';
```

Comment:
is `Inject` used anywhere?

+ what happened with spaces between symbols and brackets?

Reply:
no, it does't. Changed that and added spaces